docs/cloud/index.md

```diff
@@ -97,6 +97,11 @@ Prefect Cloud allows you to see your [events](/cloud/events/). Events provide in
 Prefect Cloud [automations](/cloud/automations/) provide additional notification capabilities beyond those in a self-hosted open-source Prefect server.
 Automations also enable you to create event-driven workflows.
 
+## Incidents <span class="badge pro"></span> <span class="badge enterprise"></span> <span class="badge beta"/>
+
+Prefect Cloud's [incidents](/concepts/incidents/)  help teams identify, rectify, and document issues in mission-critical workflows. 
+Incidents are formal declarations of disruptions to workflows, and provide a temporary control plane for collaboratively managing those disruptions. 
```

Comment:
```suggestion
Incidents are formal declarations of disruptions to a workspace. With Automations, activity in that workspace can be paused when an incident is created and resumed when it is resolved.
```

Reply:
I know we've used "temporary control plane" internally, but I find that very abstract. I think its clearer to spell it out for users.